CMakeLists.txt

```diff
@@ -580,22 +580,6 @@ include(iree_plugin_register)
 set(IREE_PACKAGE_ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}")
 set(IREE_PACKAGE_ROOT_PREFIX "iree")
 
-#-------------------------------------------------------------------------------
-# Experimental ROCM HAL driver
-# Enable with: -DIREE_EXTERNAL_HAL_DRIVERS=rocm
-#-------------------------------------------------------------------------------
-
-iree_register_external_hal_driver(
-  NAME
-    rocm
```

Comment:
https://github.com/openxla/iree/actions/runs/8665850617/job/23765392721?pr=17029#step:8:457
```
  CMake Error at runtime/src/iree/hal/drivers/CMakeLists.txt:40 (message):
    Driver passed to -DIREE_EXTERNAL_HAL_DRIVERS=rocm is not found.  Available:
    webgpu
```
That's what I was looking to see :P

Sorry, I also edited my earlier comment, the code to update to fix that error is here: https://github.com/openxla/iree/blob/40f25334d2b091a8805f7c52181ddcabd9924090/build_tools/pkgci/build_linux_packages.sh#L191

Reply:
ah I just pushed a fix for that, thanks for the catch!